patterns.go

```diff
@@ -14,6 +14,7 @@ var DNS6 = Base(ma.P_DNS6)
 
 // Define a dnsaddr, dns, dns4 or dns6 format multiaddr
 var DNS = Or(
+	Base(ma.P_DNSADDR),
```

Comment:
Ah, I forgot to update the comment when I removed this. "DNSADDR" is kind of a special beast and doesn't fit in this pattern.

The thing is... I _know_ I removed it but now I can't remember the specific problem I was trying to fix.